### PR TITLE
fix(curriculum): update object keys referenced in instructions to match provided object

### PR DESCRIPTION
**Pull request by braxex**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Because each movie object has both `Rated` and `imdbRating` key/value pairs, simply referencing `rating` could be confusing to campers. IMO explicitly referencing the keys by name (`Title` and `imdbRating`) removes ambiguity in the non-important part of this challenge and helps campers focus on learning about `.map()`. Admittedly not a mission-critical change.

### CODE REVIEW

**Review by 3zrv — APPROVED** [review]

**Review by 3zrv — DISMISSED** [review]

**Comment by ShaunSHamilton** [comment]
Hello there. Thank you, for taking the time to open this PR.

Whilst not necessarily done in the clearest way, I believe the intent behind the challenge being this way is to show the Camper the `map` method returns a completely new array, and does not need to have the same properties/structure as the original.

That is, it is common to use `map` to iterate over an array of object literals, and return a new object literal array with a different signature. So, I would not make this proposed change.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-the-map-method-to-extract-data-from-an-array.md
Comment (braxex) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-the-map-method-to-extract-data-from-an-array.md:40]:
```suggestion
The `watchList` array holds objects with information on several movies. Use `map` on `watchList` to assign a new array of objects to the `ratings` variable. Each movie in the new array should have a `title` key with the name of the film and a `rating` key with the IMDB rating. The code in the editor currently uses a `for` loop to do this, so you should replace the loop functionality with your `map` expression.
```

**Comment by braxex** [comment]
> Whilst not necessarily done in the clearest way, I believe the intent behind the challenge being this way is to show the Camper the `map` method returns a completely new array, and does not need to have the same properties/structure as the original.
> 
> That is, it is common to use `map` to iterate over an array of object literals, and return a new object literal array with a different signature. So, I would not make this proposed change.

Hmm... that's a good point, @ShaunSHamilton. Maybe it would make sense to just tweak the instruction language then and keep the expected challenge outcome the same. Something like [this suggestion I just made](https://github.com/freeCodeCamp/freeCodeCamp/pull/42613/files#r657999580):

Screen Shot 2021-06-24 at 09 28 45

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-the-map-method-to-extract-data-from-an-array.md
Comment (ShaunSHamilton) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-the-map-method-to-extract-data-from-an-array.md:40]: I think this works well 👍

**Comment by braxex** [comment]
@0xf10yd Committing my suggestion dismissed your previous review, but it also won't let me re-request review. So I'm doing it this way I guess. 🤷‍♂️ Re-review please?

**Review by 3zrv — DISMISSED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-the-map-method-to-extract-data-from-an-array.md
Comment (ShaunSHamilton) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-the-map-method-to-extract-data-from-an-array.md:40]:
```suggestion
The `watchList` array holds objects with information on several movies. Use `map` on `watchList` to assign a new array of objects to the `ratings` variable. Each movie in the new array should have only a `title` key with the name of the film, and a `rating` key with the IMDB rating. The code in the editor currently uses a `for` loop to do this, so you should replace the loop functionality with your `map` expression.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
One last change, then this LGTM.

**Review by ShaunSHamilton — APPROVED** [review]
Thank you, for making those changes. I am happy with this 👍

**Comment by braxex** [comment]
Works for me. 👍  Just need one more review and we're good to go.

**Review by gikf — APPROVED** [review]
Looks good to me, makes instructions a bit clearer. :+1:

**Pull request merged**
